web/auth/deprecated_app_list.go

```diff
@@ -85,7 +86,11 @@ func (d *DeprecatedAppList) RenderArgs(client *oauth.Client, inst *instance.Inst
 		"Favicon":     middlewares.Favicon(inst),
 		"AppName":     app.Name,
 		"Platform":    platform,
-		"StoreURL":    storeURL,
+		// template.URL is used in order to avoid the discard of url starting
+		// by `market://` (the url is replaces by "#ZgotmplZ" in case of discard).
```

Comment:
typo:

```patch
-// by `market://` (the url is replaces by "#ZgotmplZ" in case of discard).
+// by `market://` (the url is replaced by "#ZgotmplZ" in case of discard).
```